io.github.kolunmi.Bazaar.yaml

```diff
@@ -0,0 +1,247 @@
+id: io.github.kolunmi.Bazaar
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+command: bazaar-launcher
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm20
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=dri"
+  # We need directory access to system flatpak installation
+  - "--filesystem=/var/lib/flatpak"
+  # Needs to talk to host so that installing flatpaks work
+  - "--talk-name=org.freedesktop.Flatpak"
+  # Can't read flatpaks from the system without this
+  - "--system-talk-name=org.freedesktop.Flatpak.SystemHelper"
+  # Necessary so that appstream/libflatpak can read remote content, seems to entirely fail without this access
+  - "--filesystem=/var/tmp"
+  # Not having this makes it so libflatpak misbehaves and can't detect all flatpaks installed by users properly
+  # "(x) flatpaks need to be updated" when they in fact don't
+  - "--system-talk-name=org.freedesktop.Accounts"
+build-options:
```

Comment:
seems to have no top level cleanup, /include /lib/pkgconfig etc. are there aren't any?